404.html

```diff
@@ -0,0 +1,165 @@
+        <div class="box-header">
+          <h1>Looks like we are having issues with our servers!</h1>
+        </div>
+        <h3>Try refreshing in a few minutes</h3>
+        <p>You can still <a onclick='fetch("https://assets.scratch.mit.edu/83c36d806dc92327b9e7049a565c6bff.wav").then(res => res.arrayBuffer()).then(arrbuffer => {
+    let context = new AudioContext() || new webkitAudioContext;
```

Comment:
Oops...
```suggestion
    let context = new AudioContext() || new webkitAudioContext();
```

Reply:
Oh, I think it works like the current way.

Reply:
I never noticed this review before. I just fixed this in bfbe8a0 to be consistent.